02week/pigLatin.js

```diff
@@ -7,11 +7,75 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+// this will convert user input to all lowercase, and trim any whitespace
+// returns a string
+function sanitizeInput(userInput) {
+  return userInput.toLowerCase().trim();
+}
+
+// this function finds and returns the index of the first vowel in user input
+// will return -1 if no vowels are found
+function findFirstVowel(userInput) {
+  // trim input of whitespace and convert to lowercase
+  const sanitizedInput = sanitizeInput(userInput);
+  // convert input string into array of chars
+  const inputArr = sanitizedInput.split('');
+  // return variable. will contain index of first vowel found
+  let vowelIndex = -1;
+
+  // loop over user input array of chars
+  for (let i = 0; i < inputArr.length; i++) {
+    // check to see if current char is a vowel
+    // if so, set vowelIndex and break out of loop
+    if (inputArr[i] === 'a' || inputArr[i] === 'e' || inputArr[i] === 'i' || inputArr[i] === 'o' || inputArr[i] === 'u') {
```

Comment:
This is repetitive, what's a better way to do this?

Reply:
Ah! I should create an array of vowels and use the `includes` array method.

Have updated code.